contracts/cw-proposal-single/src/contract.rs

```diff
@@ -596,7 +684,7 @@ pub fn query(deps: Deps, env: Env, msg: QueryMsg) -> StdResult<Binary> {
             query_list_proposals(deps, env, start_after, limit)
         }
         QueryMsg::ProposalCount {} => query_proposal_count(deps),
-        QueryMsg::GetVote { proposal_id, voter } => query_vote(deps, proposal_id, voter),
+        QueryMsg::Vote { proposal_id, voter } => query_vote(deps, proposal_id, voter),
```

Comment:
@NoahSaso should they be supporting the v1 or v2 interface to be supported by the frontend? This module has a custom name, but is compatible with `cw-proposal-single`. Perhaps they should keep the name the same?

Reply:
Note: i branched off main before the autoclose failed execution was merged in. it included the failed proposal refund work